src/components/SelectInputNative/SelectInputNative.Overview.stories.mdx

```diff
@@ -0,0 +1,334 @@
+import { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import { Box } from '../Box/Box';
+import Card from '../Card/Card';
+import { SelectInputNative } from './SelectInputNative';
+
+<Meta
+  title="Components/SelectInputNative/Overview"
+  component={SelectInputNative}
+  parameters={{
+    controls: { hideNoControlsWarning: true },
+  }}
+/>
+
+# SelectInputNative
+
+Allows users to pick a value from a predefined list of options. Ideally, it should be used when there are 5 or more options, otherwise you should consider using a radio group instead.
+
+_NOTE:_ This is an alternative to the `SelectInput` component, but leveraging the native platform dropdown instead of a custom one. This is useful,
+when a custom UI is not needed for a select, or the native UI is preferred (this is recommended for most cases except multi-selects).
+
+## Props
+
+<ArgsTable of={SelectInputNative} />
+
+## Default
+
+All that is required to render a basic version of the SelectInputNative is a unique `id`, a `label`, a list of options passed to the `options` prop, a `value`, and an onchange event handler passed to the `onChange` prop.
+
+<Canvas>
+  <Story name="Default">
+    {() => {
+      const [value, setValue] = useState('');
```

Comment:
None of the examples in the Overview actually select an option. If you made this like react-select, then it's probably looking for an object, not a string.

Reply:
yup, just realized, I wrote the stories, refactored the API, then wrote tests but did not go back to refactor the stories. WIll get this patched up ASAP.